CMakeLists.txt

```diff
@@ -101,7 +101,8 @@ endif()
 # Dependencies
 # ============
 
-set(xeus-zmq_REQUIRED_VERSION 1.2.0)
+set(xeus_REQUIRED_VERSION 5.0.0)
```

Comment:
No need for this one, since xeus constraint is provided by either xeus-zmq or xeus-lite.

Reply:
Done.